makefile

```diff
@@ -1,4 +1,6 @@
 fmt:
```

Comment:
nit: You should rename this file to `Makefile` https://www.gnu.org/software/make/manual/make.html#Makefile-Names